packages/website/src/components/hooks/useHashState.ts

```diff
@@ -114,16 +115,69 @@ const writeStateToUrl = (newState: ConfigModel): string => {
   return '';
 };
 
+const retrieveStateFromLocalStorage = (): Partial<ConfigModel> | undefined => {
+  try {
+    const configString = window.localStorage.getItem('config');
+    if (!configString) {
+      return undefined;
+    }
+
+    const config: unknown = JSON.parse(configString);
+    if (typeof config !== 'object' || !config) {
+      return undefined;
+    }
+
+    const state: Partial<ConfigModel> = {};
+    if (hasOwnProperty('ts', config)) {
+      const ts: unknown = config.ts;
```

Comment:
Nit: no need for this type annotation:

```suggestion
      const ts = config.ts;
```